tests/test_multiraster.py

```diff
@@ -101,7 +101,9 @@ def test_stack_rasters(self, rasters) -> None:  # type: ignore
         """Test stack_rasters"""
 
         # Silence the reprojection warning for default nodata value
-        warnings.filterwarnings("ignore", category=UserWarning, message="New nodata value found in the data array.*")
+        warnings.filterwarnings(
```

Comment:
How come sometimes you have these changes in syntax, that are not related to your PR, despite the linters being enforced?

Reply:
It's related to the PR but hard to see! There is a "cell" added in the sentence for the warning :wink: